alpha/veneer/basic/basic.go

```diff
@@ -0,0 +1,81 @@
+package basic
+
+import (
+	"context"
+	"errors"
+	"os"
+	"path/filepath"
+
+	"github.com/operator-framework/operator-registry/alpha/action"
+	"github.com/operator-framework/operator-registry/alpha/declcfg"
+	"github.com/operator-framework/operator-registry/pkg/image"
+)
+
+type Veneer struct {
+	Registry image.Registry
+}
+
+func (v Veneer) Render(ctx context.Context, ref string) (*declcfg.DeclarativeConfig, error) {
+	// only taking first argument as file
+	stat, serr := os.Stat(ref)
+	if serr != nil {
+		return nil, serr
+	}
+
+	if stat.IsDir() {
+		return nil, errors.New("cannot render veneers by directory reference") // BUGBUG enhance to walk and eat all files in turn, heeding ignore file
+	}
+	return v.renderFile(ctx, ref)
+}
+
+func (v Veneer) renderFile(ctx context.Context, ref string) (*declcfg.DeclarativeConfig, error) {
+	var cfg = &declcfg.DeclarativeConfig{}
+
+	// xform any relative to absolute paths
+	abspath, err := filepath.Abs(ref)
+	if err != nil {
+		return nil, err
+	}
+	// xform to break apart dir/file elements
+	rpath, fname := filepath.Split(abspath)
+
+	root := os.DirFS(rpath)
+	file, err := root.Open(fname)
+	if err != nil {
+		return nil, err
+	}
+
+	defer file.Close()
+
+	if cfg, err = declcfg.LoadFile(root, file, ref); err != nil {
+		return cfg, err
+	}
```

Comment:
Nit:
```suggestion
	cfg, err := declcfg.LoadFile(root, file, ref)
	if err != nil {
		return cfg, err
	}
```

And drop the `var cfg = ...` declaration on line 32.